### PR TITLE
[TE] rootcause poc - collapse anomaly header on existing feedback

### PR DESCRIPTION
**Pull request by apucher**

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js [hunk 1]
```diff
@@ -116,6 +113,18 @@ export default Ember.Component.extend({
     return this.get('status') === 'NO_FEEDBACK';
   }),
 
+  isHidden: Ember.computed(
+    'isNeedFeedback',
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: nit: `needsFeedback` might be more correct (grammatically)

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js [hunk 2]
```diff
@@ -116,6 +113,18 @@ export default Ember.Component.extend({
...
+    'isHiddenUser',
+    function () {
+      const { isNeedFeedback, isHiddenUser } = this.getProperties('isNeedFeedback', 'isHiddenUser');
+      if (isHiddenUser === null) {
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: do we need to explicitely check for null (i.e. undefined should not pass? ) 
Other wise we can just do `if (isHiddenUser)`

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js [hunk 3]
```diff
@@ -130,7 +139,7 @@ export default Ember.Component.extend({
 
     toggleHidden() {
       const { isHidden } = this.getProperties('isHidden');
-      this.setProperties({ isHidden: !isHidden });
+      this.setProperties({ isHiddenUser: !isHidden });
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: you can also do `this.toggleProperty('isHidden')`

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js (hunk 1 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: yeah that's a habit of prefixing any boolean return type with 'is'. I'll rename it

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js (hunk 2 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: it's a tri-state variable - "default", "true", "false". null is the default value. spelling all of this out as an enum seems overkill

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js (hunk 3 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: the one-way interaction between isHidden and isHiddenUser is important here

**Pull request merged**